### ISSUE TITLE
Disable text wrapping on multitext TextField

### ISSUE THREAD

**Issue by IdkWhatToCallMe123** [flet-dev/flet#4132]
### Duplicate Check

- [x] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

I want to be able to disable text wrapping on multitext Textfield

**Comment by ndonkoHenri** [comment]
Flutter seems not to have a direct solution for this. 
However, with the help of [this](https://stackoverflow.com/a/63334512/) SO thread, I came up with this:
 
```py
import flet as ft


def main(page: ft.Page):
    page.add(
        ft.ListView(
            expand=True,
            horizontal=True,
            controls=[
                ft.Container(
                    width=1000,
                    bgcolor=ft.colors.BLUE,
                    content=ft.TextField(
                        hint_text="Enter your text here",
                        keyboard_type=ft.KeyboardType.MULTILINE,
                        text_vertical_align=ft.VerticalAlignment.START,
                        border=ft.InputBorder.UNDERLINE,
                        filled=True,
                        fit_parent_size=True,
                    ),
                )
            ],
        )
    )


ft.app(target=main)
```

You can't try it right now, as it makes use of a prop which will be available when #4105 gets merged.

**Comment by ndonkoHenri** [comment]
You can give the above code a try.
